_posts/2017-01-16-hierarchy-of-opinion.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+asset-type: post
+name: hierarchy-of-opinion
+title: Hierarchy of opinion
+date: 2017-01-16 16:00:00 +00:00
+author: Mashooq Badar
+image:
+    src: /assets/img/custom/blog/hierarchy-of-opinion.jpg
+tags:
+- craftsmanship
+
+---
+
+The word opinion has two distinct meanings:
+
+1. A view or judgement formed about something, not necessarily based on fact or knowledge.
+2. A statement of advice by an expert on a professional matter.
+
+Synonyms to the word opinion are: belief, judgement, thought(s), school of thought, thinking, way of thinking, mind, point of view, view, viewpoint, outlook, angle, slant, side, attitude, stance, perspective, position, standpoint etc.
```

Comment:
source for definition and synonyms are the same - adding it twice is clumsy so no direct citation needed here.